docs/developers/guides/automatic_coin_conversion.md

```diff
@@ -0,0 +1,81 @@
+<!--
+order: 8
+-->
+
+# Automated Coin Conversion
+
+Learn how to the Automated Coin Conversion feature works. {synopsis}
+
+In their ERC-20 representation, assets can be used to interact with dApps using the EVM. This standard allows developers to build applications that are interoperable with other products and services. In their Native Coin representation, they can be transferred between accounts on Evmos and other Cosmos chains using IBC. They cannot, however, be used to interact with dApps on Evmos, as Native Coins are not supported by the EVM since they don’t implement the ERC-20 standard.
+
+In order to reduce end-user complexity, Evmos should only allow single-token representation use between IBC Coin and ERC-20s. Consequently, the Evmos team developed the Automated Coin Conversion feature to achieve this goal. It consists on converting incoming IBC vouchers to ERC-20s and modifying outgoing IBC transfers to convert ERC-20s to IBC Coins. This automated conversion occurs if, and only if, the appropriate token mapping was registered through governance. If the token pair is not registered, the IBC coin will be left as is.
+
+Please read on for further understanding of this feature scope and functionality.
+
+## Outbound transactions
+
+Your users may want to move their ERC-20 tokens from Evmos onto another Cosmos chains. The automated coin conversion feature makes this operation smooth. You can send ERC-20 tokens via an IBC transfer with a single step. To do so, there is no need to make any changes on your IBC transfer logic. You only need to corroborate that the corresponding denomination is passed as a parameter. For example, if you want to transfer the ERC-20 representation of the `uosmo` token on Evmos, specifying the corresponding denomination (`Token.Denom = "uosmo"`) on the `MsgTransfer` struct will suffice. The same applies to any ERC-20 token that is not a representation of a Native Coin on other Cosmos chains. Under the hood, the protocol will atomatically make the conversion from ERC-20 token to IBC coin and perform the transfer to the desired Cosmos chain.
```

Comment:
```suggestion
Your users may want to move their ERC-20 tokens from Evmos onto other Cosmos chains. The automated coin conversion feature simplifies this operation, because it enables you to send ERC-20 tokens via an IBC transfer in a single step. To do so, there is no need to make any changes on your IBC transfer logic. You only need to ensure that the corresponding denomination is passed as a parameter. For example, if you want to transfer the ERC-20 representation of the `uosmo` token on Evmos, specifying the corresponding denomination (`Token.Denom = "uosmo"`) on the `MsgTransfer` struct will suffice. The same applies to any ERC-20 token that is not a representation of a Native Coin on other Cosmos chains. Under the hood, the protocol will automatically make the conversion from ERC-20 token to IBC coin and perform the transfer to the desired Cosmos chain.
```